### DISCUSSION TITLE
Taskbar needs a new name

### DISCUSSION THREAD

**Discussion by gavingt** [lawand-dot-io/taskbar#61] [Feature requests]
The name "Taskbar" perfectly conveys what the product is, but it's not great for SEO. Before releasing the paid version, it might be wise to change the name to something more unique. Here are a few ideas:

TaskDock
MacTasker
Taskbar Pro

**Comment by lawand-dot-io** [comment]
That's a good idea and I thought about it before. Will keep this in mind :)
